File: 7KYU/Coding 3min: Remove screws I/solution.c
#include <string.h>

unsigned sc (const char *screws)
{
   size_t len = strlen(screws);
   char last = screws[0];
   int time = 1;
   for(size_t i = 1; i < len; i++) {
     if(screws[i] == last) {
        time++;
     } else {
        last = screws[i];
        time+=6;
     }
   }

   return time + (len-1);
}

File: 7KYU/Coding 3min: Remove screws I/solution.cs
using System;

public static class Kata
{
    public static int Sc(string screws)
    {
        var len = screws.Length;
        var last = screws[0];
        var time = 1;
        for(var i = 1; i < len; i++)
        {
           if(screws[i] == last)
              time++;
           else
           {
             last = screws[i];
             time+=6;
           }
        }

        return time + (len-1);
    }
}